locale/en_US/translations.ts

```diff
@@ -2161,5 +2161,29 @@
             <source>No channels found matching your search criteria</source>
             <translation>No channels found matching your search criteria</translation>
         </message>
+        <message>
+            <source>Are You Still Watching Popup</source>
+            <translation>Are You Still Watching Popup</translation>
+        </message>
+        <message>
+            <source>How many minutes of inactivity before playback is stopped between items and a popup is shown to continue? Set to 0 to disable.</source>
+            <translation>How many minutes of inactivity before playback is stopped between items and a popup is shown to continue? Set to 0 to disable.</translation>
+        </message>
+        <message>
+            <source>Are You Still Watching?</source>
+            <translation>Are You Still Watching?</translation>
+        </message>
+        <message>
+            <source>Yes, continue</source>
+            <translation>Yes, continue</translation>
+        </message>
+        <message>
+            <source>No, stop playback</source>
+            <translation>No, stop playback</translation>
+        </message>
+        <message>
+            <source>Playback will automatically stop in 1 minute if no choice is made.</source>
+            <translation>Playback will automatically stop in 1 minute if no choice is made.</translation>
```

Comment:
```suggestion
            <translation>Playback will automatically stop in 1 minute if no buttons are pressed.</translation>
```

Reply:
If you hit a button while on this popup (up or down and I'm assuming volume as well) then it will just sit at this screen. Which is actually a nice feature if you want to use this dialog as a natural pause between episodes (get a snack, use the bathroom, etc) 😄